File: python/PROBLEMS/1179.py
x, j = 0, 0
def printImpar(impar, x):
    for i in range(0, x):
        print(f"impar[{i}] = {impar[i]}")
        
def printPar(par, j):
    for i in range(0, j):
        print(f"par[{i}] = {par[i]}")

par, impar = [], []
for i in range(0, 15):
    y = int(input())
    if y%2 == 0:
        par.append(y)
        j+=1
    else:
        impar.append(y)
        x+=1
    if x==5:
        printImpar(impar, x)
        impar = []
        x=0
    if j==5:
        printPar(par, j)
        par = []
        j=0

if x>0:
    printImpar(impar, x)
if j>0:
    printPar(par, j)
    

File: js/PROBLEMS/1179.js
var input = require('fs').readFileSync('./dev/stdin', 'utf8')
var lines = input.split('\n')
lines = lines.map(x => parseInt(x))
var impar = [5]
var par = [5]
var x=0
var j=0
for (let i=0;i<15;i++){
    let y = lines.shift()
    if (y % 2 == 0){
        par[j] = y;
        j++;
    }
    else{
        impar[x] = y;
        x++;
    }
    if (x==5){
        printImpar(impar, x);
        x=0;
        var impar = []
    }
    if (j==5){
        printPar(par, j);
        j=0;
        var par = [5];
    }

}
if (x>0){
    printImpar(impar, x);
}
if (j>0){
    printPar(par, j);
}

function printPar(par, j){
    for(let i=0;i<j;i++){
        console.log(`par[${i}] = ${par[i]}`)
    }
}
function printImpar(impar, x){
    for(let i=0;i<x;i++){
        console.log(`impar[${i}] = ${impar[i]}`)
    }
}